docs/manual_span_attributes.md

```diff
@@ -4,84 +4,108 @@ This document contains the list of manual Span Attributes used throughout the de
 ## CurrencyService
 
-* `app.currency.conversion.from`
-* `app.currency.conversion.to`
+| Name                           | Type   | Description                   |
+|--------------------------------|--------|-------------------------------|
+| `app.currency.conversion.from` | string | Currency code to convert from |
+| `app.currency.conversion.to`   | string | Currency code to convert to   |
 
 ## EmailService
 
-* `app.email.sent`
-* `app.order.id`
-* `app.shipping.cost.currency`
-* `app.shipping.cost.total`
-* `app.shipping.tracking.id`
+| Name                         | Type   | Description                       |
+|------------------------------|--------|-----------------------------------|
+| `app.email.sent`             | string | Email used for order confirmation |
+| `app.order.id`               | string | Order Id                          |
+| `app.shipping.cost.currency` | string | Order currency                    |
+| `app.shipping.cost.total`    | string | Order cost total                  |
+| `app.shipping.tracking.id`   | string | Order shipping tracking Id        |
 
 ## FeatureFlagService
 
-* None yet
+| Name      | Type | Description |
+|-----------|------|-------------|
+| None yet  |      |             |
 
 ## Frontend
 
-* `app.cart.size`
-* `app.cart.items.count`
-* `app.cart.shipping.cost`
-* `app.cart.total.price`
-* `app.currency`
-* `app.currency.new`
-* `app.order.total`
-* `app.product.id`
-* `app.product.quantity`
-* `app.products.count`
-* `app.request.id`
-* `app.session.id`
-* `app.user.id`
+| Name                     | Type   | Description                   |
+|--------------------------|--------|-------------------------------|
+| `app.cart.size`          | number | Total number of items in cart |
+| `app.cart.items.count`   | number | Count of unique items in cart |
+| `app.cart.shipping.cost` | number | Cart shipping cost            |
+| `app.cart.total.price`   | number | Cart total price              |
+| `app.currency`           | string | User currency                |
+| `app.currency.new`       | string | New currency to set           |
+| `app.order.total`        | number | Order total cost              |
+| `app.product.id`         | string | Product Id                    |
+| `app.product.quantity`   | number | Product quantity              |
+| `app.products.count`     | number | Total products displayed      |
+| `app.request.id`         | string | Request Id                    |
+| `app.session.id`         | string | Session Id                    |
+| `app.user.id`            | string | User Id                       |
 
 ## LoadGenerator
 
-* None yet
+| Name      | Type | Description |
+|-----------|------|-------------|
+| None yet  |      |             |
 
 ## PaymentService
 
-* `app.payment.cost`
-* `app.payment.currency`
+| Name                   | Type   | Description        |
+|------------------------|--------|--------------------|
+| `app.payment.cost`     | number | Total payment cost |
```

Comment:
I'm not expert in payment systems, I wonder if this is normally called "amount (transaction amount)" or "cost"?

Reply:
I agree this should be called `amount` and not `cost`. 

Next week I will propose changes to all attributes so we can remove unnecessary ones, normalize names, as well as make slight name adjustments where required. We can track that in with #280